hieradata_aws/class/exception_handler.yaml

```diff
@@ -0,0 +1,28 @@
+---
+
+govuk::node::s_base::apps:
+  - errbit
+
+lv:
+  mongodb:
+    pv:
+      - '/dev/xvdg'
+    vg: 'backup'
+  data:
+    pv: '/dev/xvdf'
+    vg: 'mongodb'
+
+mount:
+  /var/lib/mongodb:
+    disk: '/dev/mapper/mongodb-data'
+    govuk_lvm: 'data'
+    mountoptions: 'defaults'
+  /var/lib/automongodbbackup:
+    disk: '/dev/mapper/backup-mongodb'
+    govuk_lvm: 'mongodb'
+    mountoptions: 'defaults'
+
+mongodb::firewall::enable_firewall_rules: false
+
+mongodb::server::replicaset_members:
+  'errbit.issam.integration.govuk-internal.digital':
```

Comment:
Will this work? If this is doing a lookup of the ELB DNS name then will it try to resolve the IP of the ELB, rather than resolve the address of the local server? It will probably have issues trying to route out and then back in from it's own load balancer (unless AWS works differently to what I'm used to).

It may be easier to specify "localhost".